lib/fs.js

```diff
@@ -1585,6 +1585,10 @@ function encodeRealpathResult(result, options, err) {
   }
 }
 
+// This is removed from the fs exports in lib/module.js in order to make
+// sure that this stays internal.
+const realpathCacheKey = fs.realpathCacheKey = Symbol('realpathCacheKey');
```

Comment:
There is no `internal/fs` because using that would break old graceful-fs versions… that’s what #6413 is all about.

It’s not meant to be public, and as the comment above indicates (if you have suggestions on how to clarify, that would be welcomed!), `realpathCacheKey` is removed from the exports of `fs` automatically. It’s a bit hacky, yes.